scripts/mvftoms.py

```diff
@@ -545,31 +540,38 @@ def _cp_index(a1, a2, pol):
         field_names, field_centers, field_times = [], [], []
         obs_modes = ['UNKNOWN']
         total_size = 0
-
-        # Create the MeasurementSet
-        table_desc, dminfo = ms_extra.kat_ms_desc_and_dminfo(
+        dump_start = 0
+        if os.path.exists(ms_name):
+            print(f"MS '{ms_name}' already exists - continuing...")
+            nbaselines = dataset.shape[2]/4
+            nrows = ms_extra.table_rows(ms_name, verbose=options.verbose)
+            dump_start = int(nrows // nbaselines)
```

Comment:
`nbl` had better divide exactly into `nrows`... So no need for the `int` cast (I realise it's there because `nbaselines` was `float`).

I would also do some more spot checks of the existing MS, like the actual channels and baselines involved. It is possible that the second `mvftoms` run uses different script options that will result in an incompatible MS even though the MVF part (and CBID) is identical.

Or we can store these script options inside the MS somehow - that would be cool 🙂 The HISTORY sub-table might be the right place for this.